src/components/Button/README.md

```diff
@@ -373,6 +373,16 @@ Use when there is only one primary action but other related actions can be taken
 </div>
 ```
 
+### Button with Icon
+
+By default icons are placed before the button content, use the `iconPlacement` property to display the icon after the content instead.
+
+```jsx
+<Button iconPlacement="after" icon={ExternalMinor}>
+  Hello
+</Button>
+```
+
```

Comment:
We should probably have a default button with icon example as well. Surprised we didn't have one before